lmdeploy/lite/utils/load.py

```diff
@@ -4,8 +4,8 @@
 from accelerate import infer_auto_device_map, init_empty_weights
 from transformers import AutoConfig, AutoModelForCausalLM
 
+from lmdeploy.legacy.pytorch.model import LoadWoInit
```

Comment:
avoid using function in legacy.
There is `LoadNoInit` in `lmdeploy.pytorch.accel`

Reply:
Done!

Reply:
Done!